charts/timescaledb-single/templates/configmap-pgbouncer.yaml

```diff
@@ -48,5 +41,13 @@ data:
     client_tls_sslmode=require
     client_tls_key_file = /etc/certificate/tls.key
     client_tls_cert_file = /etc/certificate/tls.crt
+    # Loading these last, to allow for overriding of any of the above defaults. Since the 
+    # last instance of an option gets used.
+    # DANGER: The above settings are considered to be essential to how this helm chart works. 
```

Comment:
```suggestion
```

Should we document this in the `values.yaml`?  I don't expect anyone to have a look at the templates themselves

Reply:
That's a good point. We should probably document "expected" values.yaml values as well as "advanced / dangerous" ones.

I'll make some changes there as well.

Reply:
I've added instructions and a warning to the values.yaml file, and cleaned up the template.